### PR TITLE
Core, SQL, Scripts, Documentation: Updated zone list.

### PR DESCRIPTION
**Pull request by TeoTwawki**
Believe it or not, this is retail aside from missing the NPC positional data.

### CODE REVIEW

File: scripts/zones/GM_Home/npcs/Janken.lua [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+    player:starteEvent(139);
```
Comment (xdemolish) [inline scripts/zones/GM_Home/npcs/Janken.lua]: STARTE
EVENTOOOO

Comment (xdemolish) [inline scripts/zones/GM_Home/npcs/Janken.lua]: it's in your other luas too

**Comment by TeoTwawki** [comment]
@xdemolish Amended

**Comment by xdemolish** [comment]
comment out the npcs too (since their pos data is all 0'd anyways)

**Comment by TeoTwawki** [comment]
Done.

**Pull request merged**